src/Proto3/Wire/Encode.hs

```diff
@@ -504,7 +561,7 @@ shortText num = embedded num . MessageBuilder . RB.shortTextUtf8
 -- >>> 1 `byteString` "testing"
 -- Proto3.Wire.Encode.unsafeFromLazyByteString "\n\atesting"
 byteString :: FieldNumber -> B.ByteString -> MessageBuilder
-byteString num = embedded num . MessageBuilder . RB.byteString
+byteString !num = embedded num . MessageBuilder . RB.byteString
```

Comment:
Perhaps replace `MessageBuilder . RB.byteString` with `unsafeFromByteString`?

Reply:
Sure.  A bit more work for the compiler when expanding, but not a significant amount probably.